backend/infrahub/database/__init__.py

```diff
@@ -489,7 +489,14 @@ async def validate_database(
 
 
 async def get_db(retry: int = 0) -> AsyncDriver:
-    URI = f"{config.SETTINGS.database.protocol}://{config.SETTINGS.database.address}:{config.SETTINGS.database.port}"
+    base_uri = (
+        f"{config.SETTINGS.database.protocol}://{config.SETTINGS.database.address}:{config.SETTINGS.database.port}"
+    )
+
+    if config.SETTINGS.database.policy is not None:
+        URI = f"{base_uri}?policy={config.SETTINGS.database.policy}"
+    else:
+        URI = base_uri
```

Comment:
I think all of this should be moved to the config and the `DatabaseSettings` class where we can have a property for database_uri that combines the above logic and returns a string.